google/datalab/utils/_http.py

```diff
@@ -44,17 +44,15 @@ def __init__(self, status, content):
     self.message = 'HTTP request failed'
     # Try extract a message from the body; swallow possible resulting ValueErrors and KeyErrors.
     try:
-      if type(content) == str:
-        error = json.loads(content)['error']
-      else:
-        error = json.loads(str(content, encoding='UTF-8'))['error']
+      error = json.loads(content, encoding='UTF-8')['error']
```

Comment:
Why make this change? It seems the old if/else is there for py2,3 compatibility.

Reply:
The old code did not attempt to decode the content in python3 and the change was necessary. However, the json loads api handles all cases of string, bytes and basestring and correctly ignores the encoding flag in python3 if it doesn't need it.

I can revert the change or add new test cases to make sure that this implementation works for relevant python versions we need to support.